### DISCUSSION TITLE
slope value always at 1

### DISCUSSION THREAD

**Discussion by Emisanduc** [NightscoutFoundation/xDrip#2440] [Q&A]
I had to reinstall the application and since then in the calibration graph the slope is always at 1 and the intercept is between -50 and -140 depending on the calibration. I'm having a hard time keeping xdrip calibrated so the functionality of avoiding the hair test was lost. I would like to know how I can solve the problem. From already thank you very much
![33BB73D9-046F-4CF7-8C1A-EE09E57CF7B4](https://user-images.githubusercontent.com/96702567/200467552-26c9e483-a422-41c6-ad61-ac2e855c130b.jpeg)
![DB6308B9-1C53-451D-B7E3-2F2DA13A1D43](https://user-images.githubusercontent.com/96702567/200467563-5efb9213-54cc-4672-89a2-af823da4c97d.jpeg)

**Comment by Navid200** [comment]
@psonnera

**Comment by Navid200** [comment]
@Emisanduc Would you please clarify what the data source is?  You can post a screenshot of the following page:  Settings &#8722;> Hardware Data Source

**Reply by Emisanduc** [reply]
Thank you very much for the help, I sent a photo of how the hardware data is configured

![image](https://user-images.githubusercontent.com/96702567/200687160-cb02c599-ffce-48b3-bbe5-572ae313cd33.jpg)

**Comment by psonnera** [comment]
Do you have non fixed calibration slope disabled by any chance? https://xdrip.readthedocs.io/en/latest/calibrate/advancedcal/#libre-specific

**Comment by Emisanduc** [comment]
Enviado desde mi iPhone

**Reply by Navid200** [reply]
Please don't post using an email client.
Please respond to psonnera.  He may be able to help you.
